livekit-android-sdk/src/main/java/io/livekit/android/room/Room.kt

```diff
@@ -635,8 +636,18 @@ constructor(
     /**
      * @suppress
      */
-    override suspend fun onFullReconnect() {
-        localParticipant.republishTracks()
+    override suspend fun onPostReconnect(isFullReconnect: Boolean) {
+        if (isFullReconnect) {
+            localParticipant.republishTracks()
+        } else {
+            val remoteParticipants = remoteParticipants.values.toList()
+            for (participant in remoteParticipants) {
+                for (pub in participant.videoTracks.values) {
```

Comment:
`UpdateTrackSettings` has a `disabled` field. I am guessing that is set for audio tracks when subscriber mutes the remote audio track. So, I think this should include update on both audio & video tracks?

Reply:
Hmm, at the moment the server doesn't seem to be using the disabled field for audio tracks. Not sure if we're meaning to use UpdateTrackSettings for audio tracks (though it does make sense for it to.)

Reply:
It should be @davidliu . If not, it is a bug.

It applies to both audio and video (the naming could be confusing) - https://github.com/livekit/livekit-server/blob/745939ec24c32562c7c40dfcbbb9fb53625edab8/pkg/rtc/subscribedtrack.go#L90. It calls `updateDownTrackMute` here (https://github.com/livekit/livekit-server/blob/745939ec24c32562c7c40dfcbbb9fb53625edab8/pkg/rtc/subscribedtrack.go#L101) which will stop the forwarder for that track in the SFU.